clusters/app.ci/prow-monitoring/Makefile

```diff
@@ -1,147 +1,20 @@
-	oc delete configmap -n prow-monitoring grafana-dashboard-definitions-0 --ignore-not-found=true --as=system:admin
-	oc delete configmap -n prow-monitoring grafana-dashboards --ignore-not-found=true --as=system:admin
-	oc delete configmap -n prow-monitoring grafana-datasources --ignore-not-found=true --as=system:admin
-	oc delete secret -n prow-monitoring grafana-config --ignore-not-found=true --as=system:admin
-	oc process -f ./grafana_expose.yaml | oc delete --ignore-not-found=true --as=system:admin -f -
-	oc delete -f ./grafana_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./client_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./ci_hook_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./prometheus_project.yaml --wait=true --ignore-not-found=true --as=system:admin
-
-
-.PHONY : grafana-debug-deploy
-grafana-debug-deploy: grafana-debug-generate-k8s-dashboards generate-mixins
-	$(eval grafana_password := $(shell uuidgen))
-	@echo "Running grafana-debug-deploy ... with password: $(grafana_password)"
-	# oc new-project prow-monitoring-stage --skip-config-write=true
-	# oc adm policy add-role-to-group admin ci-admins --as system:admin -n prow-monitoring-stage
-	oc get project prow-monitoring-stage
-	oc create -n prow-monitoring-stage configmap grafana-config --from-file=./debug/grafana.ini --dry-run -o yaml | oc apply -f -
-	@echo "replacing the password in ./debug/grafana_secret.yaml ..."
-	@sed -i -e "s|CHANGE_ME|$(grafana_password)|g" ./debug/grafana_secret.yaml
-	oc apply -f ./debug/grafana_secret.yaml
-	git checkout ./debug/grafana_secret.yaml
-	oc get secret -n openshift-monitoring --as system:admin grafana-datasources -o json | jq -r '.data["prometheus.yaml"]' | base64 -d | jq '.datasources[].name = "prometheus-k8s"' > /tmp/prometheus-k8s.yaml
-	oc create -n prow-monitoring-stage configmap grafana-datasources --from-file=datasources.yaml=./debug/grafana_datasources.yaml --from-file=prometheus-k8s.yaml=/tmp/prometheus-k8s.yaml --dry-run -o yaml | oc apply -f -
-	rm -fv /tmp/prometheus-k8s.yaml
-	oc delete pod -n prow-monitoring-stage -l app=grafana --ignore-not-found=true
-	oc apply -f ./debug/grafana_deploy.yaml
-	@echo "admin password: $$(oc get secret -n prow-monitoring-stage grafana -o jsonpath='{.data.password}' | base64 -d)"
-
-.PHONY : grafana-debug-generate-k8s-dashboards
-grafana-debug-generate-k8s-dashboards:
-	make -C ./mixins generate-k8s-dashboards
-
-.PHONY : grafana-debug-cleanup
-grafana-debug-cleanup:
-	@echo "Running grafana-debug-deploy ..."
-	oc get project prow-monitoring-stage
-	oc delete -f ./debug/grafana_deploy.yaml --ignore-not-found=true
-	oc create -n prow-monitoring-stage configmap grafana-config --from-file=./debug/grafana.ini --dry-run -o yaml | oc delete --ignore-not-found=true -f -
-	oc delete -n prow-monitoring-stage configmap grafana-datasources --ignore-not-found=true
 
+.PHONY : all
+all:
```

Comment:
This doesn't do anything or am I missing sth?

Reply:
Yes. It does not apply anything.
Sometimes, I do `make` and forget the target. Then I will run the first target (which could do some undesired).
This one would avoid the mistake like that.

Reply:
Ah. Its confusing though to have a target named `all` which doesn't do anything. Mind renaming it to something like `no-op` or just `default`?
Also, the canonical way to do this is to set the `DEFAULT_GOAL` target, it will remain the default even if someone puts a new target in front of  yours: https://www.systutorials.com/how-to-change-the-default-target-of-make/

Reply:
Updated.